Saturation/PredicateSplitPassiveClauseContainer.cpp

```diff
@@ -44,17 +48,27 @@ PredicateSplitPassiveClauseContainer::PredicateSplitPassiveClauseContainer(bool
     USER_ERROR("Queue " + name + ": The number of cutoffs needs to match the number of queues, but " + Int::toString(_cutoffs.size()) + " != " + Int::toString(_queues.size()));
   }
 
+  if (_randomize) {
```

Comment:
I trust you know what you're doing here, I've no idea what `PredicateSplitPassiveClauseContainer` does!

Reply:
It's a meta-container with its own ratio. The change is in that it may hold more than two containers, so that ratio is a tuple of things numbers not just a pair